src/plugins/discover/public/application/angular/discover_state.ts

```diff
@@ -191,6 +195,26 @@ export function getState({
   };
 }
 
+export function getStateDefaults(
```

Comment:
More clashes with my PR 😂 Joke aside, is this change really needed for this PR?

Reply:
good point, should not be necessary because of #79062 ... so this should be merged first

Reply:
@majagrubic removed this change, since not necessary